gallery/src/gallery-components/GalleryTiles.tsx

```diff
@@ -6,24 +6,27 @@
  */
 import React, { FunctionComponent, ReactNode } from 'react';
 import classnames from 'classnames';
-import { ensureArray } from '@sonatype/react-shared-components/util/jsUtil';
+import { ensureArray } from '../util/jsUtil';
 
 import CodeExample, { Props as CodeExampleProps } from '../CodeExample';
 
 interface PropsWithRequiredChildren {
   children: ReactNode;
 }
 
-// Props for GalleryTile
-interface GalleryTileProps extends PropsWithRequiredChildren {
+type GalleryBaseProps = {
   title: string;
   className?: string;
 }
 
+// Props for GalleryTile
+type GalleryTileProps = PropsWithRequiredChildren & GalleryBaseProps;
+
 type StringOrCodeExampleProps = string | CodeExampleProps;
 
-interface GalleryExampleTileProps extends GalleryTileProps {
-  description: string;
+interface GalleryExampleTileProps extends GalleryBaseProps {
+  children?: ReactNode;
+  description: ReactNode;
```

Comment:
Realized while writing these that the description often would benefit from having HTML elements in it, particularly `<code>` elements.